src/test/java/net/revelc/code/impsort/FileTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.revelc.code.impsort;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.Test;
+
+/**
+ * Test class for special file cases.
+ */
+public class FileTest {
+
+  private static Grouper eclipseDefaults =
+      new Grouper("java.,javax.,org.,com.", "", false, false, true);
+
+  /**
+   * Test successfully parsing empty (0 byte) file.
+   */
+  @Test(expected = EmptyFileException.class)
```

Comment:
I believe the latest JUnit supports `assertThrows`, which is better than having expected params here, because you can use it on a specific line of code. You can also get a reference to the exception caught, to do additional checks, like `getPath`

Reply:
Yes, I understand. I will change it.